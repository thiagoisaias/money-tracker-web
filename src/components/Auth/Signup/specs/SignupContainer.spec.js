import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { SignupContainer } from "../SignupContainer";

describe("Signup FormContainer", () => {
  const mockProps = {
    error: null,
    onClearAuthError: jest.fn(),
    onSignup: jest.fn(),
    isLoading: false,
    history: {}
  };
  const component = shallow(<SignupContainer {...mockProps} />);

  it("renders properly", () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
