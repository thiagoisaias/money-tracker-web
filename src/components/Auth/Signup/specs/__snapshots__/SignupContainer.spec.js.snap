// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup FormContainer renders properly 1`] = `
<Component
  error={null}
  isLoading={false}
  onSignup={[Function]}
  onSubmitData={[Function]}
/>
`;
